mlflow/keras.py

```diff
@@ -476,8 +486,21 @@ def autolog():
 
     If training does not end due to early stopping, then ``stopped_epoch`` will be logged as ``0``.
 
-    MLflow will also log the parameters of the EarlyStopping callback,
+    MLflow will also log the parameters of the ``EarlyStopping`` callback,
     excluding ``mode`` and ``verbose``.
+
+    .. code-block:: python
```

Comment:
If this example doesn't include `EarlyStopping`, we should either move it above `EarlyStopping` info or include `EarlyStopping` into this example.

Reply:
Agreed. will move above it, and later I'll include EarlyStopping examples, and also extend TF examples T1 and T2 to include or extend with EarlyStopping. For this PR, let's sort this one out so we can close this issue.

WDYT?